core/extension/gdextension_manager.cpp

```diff
@@ -281,6 +354,8 @@ void GDExtensionManager::_bind_methods() {
 	BIND_ENUM_CONSTANT(LOAD_STATUS_NEEDS_RESTART);
 
 	ADD_SIGNAL(MethodInfo("extensions_reloaded"));
+	ADD_SIGNAL(MethodInfo("extension_loaded"));
+	ADD_SIGNAL(MethodInfo("extension_unloading"));
```

Comment:
The arguments passed to these signals should be given here, for example (untested):

```suggestion
	ADD_SIGNAL(MethodInfo("extension_loaded", PropertyInfo(Variant::OBJECT, "extension", PROPERTY_HINT_RESOURCE_TYPE, "GDExtension"));
	ADD_SIGNAL(MethodInfo("extension_unloading"), PropertyInfo(Variant::OBJECT, "extension", PROPERTY_HINT_RESOURCE_TYPE, "GDExtension"));
```

Reply:
Oh thanks! Was not sure how to defined the "GDExtension" type. expect for a little `)` error, this code worked perfectly. Tested and documentation updated.
Done!